### DISCUSSION TITLE
tweepy examples

### DISCUSSION THREAD

**Discussion by ihabpalamino** [tweepy/tweepy#2143] [Questions] [answered]
### Question

hello guys i am new at using tweepy can i have some examples of scrapring tweets and their number of likes reteweet comments date of post... of otherr accounts by using username and the since until date Thanks

### Checklist

- [X] I have searched for duplicate questions.
- [ ] If applicable, I have removed any visible credentials from my code and/or screenshots.

**Comment by Harmon758** [comment, marked as answer]
There are [examples in the documentation](https://tweepy.readthedocs.io/en/v4.14.0/examples.html).

**Reply by ihabpalamino** [reply]
i am sorry i have not found an example how to scrape datas of other account that is not mine by it username

**Reply by Harmon758** [reply]
Tweepy does not scrape data. It uses Twitter's API.
There's examples of getting users as well as their Tweets on that page.
